src/components/Login/LoginForm/index.tsx

```diff
@@ -0,0 +1,137 @@
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  AlertTitle,
+  Box,
+  Button,
+  Flex,
+  Input,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
+import { useAuth } from "../../../hooks/useAuth";
+import useFetch from "../../../hooks/useFetch";
+
+const SignUpButton = styled.button`
+  color: #e2e8f0;
+  text-decoration: underline;
+  text-align: center;
+  display: block;
+  margin: 0 auto;
+  margin-top: 60px;
+  border-radius: 1px;
+`;
+
+interface InputProps {
+  value: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+interface LoginFormProps {
+  idInput: InputProps;
+  pwInput: InputProps;
+  toggleSignUpModal: () => void;
+}
+
+const LoginForm: React.FC<LoginFormProps> = ({
+  idInput,
+  pwInput,
+  toggleSignUpModal,
+}) => {
+  const [loginError, setLoginError] = useState("");
+  const { setToken } = useAuth();
+  const navigate = useNavigate();
+
+  const { result, loading, statusCode, refresh, error } = useFetch({
```

Comment:
```suggestion
  const { result, loading, statusCode, refresh: startLogin, error } = useFetch({
```

이렇게 작성하면, 67번째 줄처럼 불필요하게 함수를 따로 wrapping 할 필요도 없고, 직관적으로 변수명을 재할당할 수 있어서 좋습니다.